core/unit_test/TestExecSpaceThreadSafety.hpp

```diff
@@ -95,6 +95,11 @@ TEST(TEST_CATEGORY, exec_space_thread_safety_range) {
 #ifdef KOKKOS_ENABLE_OPENMPTARGET
   if (std::is_same_v<TEST_EXECSPACE, Kokkos::Experimental::OpenMPTarget>)
     GTEST_SKIP() << "skipping since test is known to fail for OpenMPTarget";
+#endif
+// FIXME_SYCL
+#ifdef KOKKOS_ENABLE_SYCL_OUT_OF_ORDER_QUEUES
```

Comment:
(Optional)
```suggestion
#ifdef KOKKOS_ENABLE_SYCL_OUT_OF_ORDER_QUEUES  // FIXME_SYCL
```
Same comment everywhere.
It comes handy if you grep FIXME_SYCL in the code base.
You get more information without having to jump to the matches.